BruteForce/0504/[백준] 2961 - 도영이가 만든 맛있는 음식.py

```diff
@@ -0,0 +1,50 @@
+# 넉넉하게 잡아도 시간복잡도는  O(n^2)  100만이 넘지 않는다 > 완전탐색
+# 쓴맛을 1~ N개 까지 선택하는 조합 구해서 각 경우 다 곱하기
+# 단맛을 1~N개 까지 선택하는 조합 구해서 각 경우 다 더하기
+
+# 자꾸 틀려서 이유가 뭘까 생각해봤는데 재료를 각각 사용하는것으로 착각했다.
+# 4
+# 1 7
+# 2 6
+# 3 8
+# 4 9 면 1, 2, 3, 4, 1 2, 1 3,...      7, 6, 8, 9, 7 6, 7 8, 모두 각각의 경우의수를 다 구해야하는 줄 알았다.
+# 하지만 그게 아니었고 재료를 사용할 때 한 줄은 한번에 사용하는 것이었다.
+
+# 조합을 이용해서 구하면 되는 문제
+from itertools import combinations
+
+N = int(input())
+taste = [list(map(int,input().split())) for _ in range(N)]
+bitter=[] 
+sweet=[]
+for a,b in taste:   #쓴맛과 단맛에 요소 나누어 담기
+    bitter.append(a)
+    sweet.append(b)
+
+min_taste=1000000000       #임의의 최대값
+
+for i in range(len(bitter)):   # 하나씩만 선택해서 최소값구하는경우
+    min_taste=min(min_taste,abs(bitter[i]-sweet[i]))  
```

Comment:
처음에 재료값 입력 받을 때 바로 `min_taste`를 구하는게 더 코드가 짧지 않을까요??
```suggestion
min_taste = 1e9
bitter, sweet = [], []
for _ in range(N):
    a, b = map(int, input().split())
    min_taste = min(min_taste, abs(a-b))
    bitter.append(a)
    sweet.append(b)
```

Reply:
네네! 1000000000 에요! `min_taste=1000000000` 한것처럼, 충분히 큰 수로 설정했어요! int(1e9) 로 해서 int로 변환해서 쓰거나, 아니면 그냥 1e9로 하게되더라구요.